### DISCUSSION TITLE
How do you run this with the Steam version?

### DISCUSSION THREAD

**Discussion by IndigoAK** [LostArtefacts/TRX#297] [Q&A]
Running this with the naked Steam version results in an error about not being able to load a .phd file.  The Steam version is packed and doesn't contain any of the loose files you mention in the readme.  Is there an extra step needed with the Steam version not documented?

**Comment by aredfan** [comment]
[https://www.pcgamingwiki.com/wiki/Tomb_Raider_(1996)](https://www.pcgamingwiki.com/wiki/Tomb_Raider_(1996))

Quoted from the '**Install Tomb Raider Retail Fix**' section:

> For the GOG.com and Steam versions, extract GAME.GOG with [UniExtract](https://github.com/Bioruebe/UniExtract2/releases) to get these folders.

This step should give you the DATA and FMV folders.

**Comment by rr-** [comment]
There will be an installer in #529 that simplifies this process a lot.
